sky/build/sky_precompilation_sdk.gni

```diff
@@ -92,4 +94,30 @@ template("sky_precompilation_sdk") {
       ":$copy_config_gen_target_name",
     ]
   }
+
+  sdk_zip_gen_target_name = target_name + "_zip"
+  action(sdk_zip_gen_target_name) {
+    zip_path = "$root_out_dir/$sdk_name.zip"
+    script = "//sky/build/zip_xcodeproj.py"
+    depfile = "$target_gen_dir/$sdk_zip_gen_target_name.d"
+    outputs = [
+      depfile,
+      zip_path,
+    ]
+    args = [
+      "--depfile",
+      rebase_path(depfile, root_build_dir),
+      "--input",
+      rebase_path(sdk_dir),
```

Comment:
This rule is similar to the copy rule above. I considered individually listing the files. But the Xcode project contains a lot of bundles (projects, workspaces, assets, iconsets, etc.). Individually including all these files would require a separate GN copy target for each bundle/directory (so I can set the base path for each). While this ensures complete correctness, it would also wipe changes I make to the project (setting FLUTTER_APPLICATION_PATH, FLUTTER_ROOT, selecting my provisioning profile, etc.) while I am iterating on an iOS build.

In its current state `sdk_zip_gen_target_name` still lists its dependencies explicitly via the `copy_all_gen_target_name` group. So the tools, binaries, snapshotters, etc.. would still always be up to date.

I could completely disregard the contents of the copy phase and specify the items again when I zip up the project. But that seems like repeating a lot of same things again.